Jenkinsfile

```diff
@@ -1,4 +1,45 @@
 #!groovy
+
+void printInfo() {
```

Comment:
I tried to move this to a separate file (which might be possible, but let's discuss first if we like the approach)